internal/ansible/runner/runner.go

```diff
@@ -366,6 +366,9 @@ func (r *runner) makeParameters(u *unstructured.Unstructured) map[string]interfa
 
 	specKey := fmt.Sprintf("%s_spec", objKey)
 	parameters[specKey] = spec
+	if r.markUnsafe {
+		parameters[specKey] = markUnsafe(spec)
+	}
```

Comment:
@gaelgatelement Would you mind elaborating a bit more why we need to mark the whole spec as unsafe?

Reply:
That's a rather old issue, so I'll describe it a bit from memory. In our CRD we have some string properties which receive Jinja templating. Marking the whole spec as unsafe does not have much side effect (that we could see, we are using this feature for months in our fork now), while it avoids issues with this jinja templating.